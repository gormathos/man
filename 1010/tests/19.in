sol_id;user_id;task_id;lang_id;score
151555;1092;2003;3;29.17
151556;1027;2001;1;0.00
151558;1038;2001;5;100.00
151561;1057;2004;3;18.37
151565;1048;2005;4;7.14
151570;1017;2003;2;83.33
151576;1067;2003;3;0.00
151583;1055;2003;4;0.00
151591;1027;2001;1;0.00
151600;1029;2001;4;16.67
151610;1070;2003;1;100.00
151621;1099;2003;5;100.00
151633;1052;2003;3;39.58
151646;1007;2005;3;16.67
151660;1015;2001;3;100.00
151675;1001;2005;5;4.76
151691;1014;2002;3;92.50
151708;1002;2001;1;63.33
151726;1028;2005;3;100.00
151745;1027;2002;4;2.50
151765;1016;2002;4;20.00
151786;1093;2005;1;100.00
151808;1037;2003;3;75.00
151831;1096;2003;1;100.00
151855;1021;2004;2;67.35
151880;1078;2004;5;0.00
151906;1044;2002;1;50.00
151933;1048;2003;1;20.83
151961;1060;2005;5;100.00
151990;1006;2003;2;100.00
152020;1041;2001;4;100.00
152051;1089;2001;2;36.67
152083;1037;2005;5;0.00
152116;1002;2003;5;41.67
152150;1062;2001;3;100.00
152185;1026;2001;1;16.67
152221;1037;2001;4;23.33
152258;1046;2004;2;20.41
152296;1055;2002;5;47.50
152335;1027;2002;3;0.00
152375;1025;2005;4;16.67
152416;1005;2001;3;100.00
152458;1051;2005;1;0.00
152501;1086;2004;1;0.00
152545;1098;2004;1;2.04
152590;1086;2004;3;40.82
152636;1024;2005;3;73.81
152683;1078;2005;2;2.38
152731;1081;2002;3;15.00
152780;1039;2001;1;0.00
152830;1050;2005;1;0.00
152881;1094;2004;2;100.00
152933;1055;2004;4;100.00
152986;1040;2005;4;100.00
153040;1032;2005;3;0.00
153095;1050;2003;4;72.92
153151;1085;2003;2;62.50
153208;1093;2005;4;100.00
153266;1088;2002;5;100.00
153325;1097;2004;3;100.00
153385;1029;2003;1;66.67
153446;1075;2005;2;90.48
153508;1069;2001;4;36.67
153571;1036;2004;5;0.00
153635;1086;2004;2;0.00
153700;1018;2002;1;0.00
153766;1091;2003;1;45.83
153833;1004;2003;5;100.00
153901;1083;2005;4;61.90
153970;1056;2002;2;12.50
154040;1053;2004;3;100.00
154111;1023;2005;3;100.00
154183;1047;2005;2;100.00
154256;1018;2004;3;100.00
154330;1065;2001;3;100.00
154405;1077;2003;3;100.00
154481;1038;2001;2;23.33
154558;1045;2005;3;100.00
154636;1087;2003;1;100.00
154715;1096;2004;5;51.02
154795;1097;2001;4;0.00
154876;1009;2004;1;2.04
154958;1038;2001;3;0.00
155041;1007;2003;2;0.00
155125;1084;2004;1;100.00
155210;1089;2003;5;100.00
155296;1004;2003;3;100.00
155383;1089;2003;2;47.92
155471;1091;2003;3;81.25
155560;1076;2002;5;2.50
155650;1064;2004;5;0.00
155741;1062;2001;3;0.00
155833;1087;2001;1;56.67
155926;1061;2003;2;0.00
156020;1082;2002;2;75.00
156115;1096;2004;2;95.92
156211;1053;2005;4;0.00
156308;1050;2003;3;37.50
156406;1053;2004;5;0.00
156505;1055;2001;2;96.67
156605;1030;2004;5;100.00
156706;1035;2001;2;0.00
156808;1016;2002;5;100.00
156911;1017;2004;2;28.57
157015;1055;2004;5;0.00
157120;1095;2004;2;100.00
157226;1008;2001;4;80.00
157333;1057;2003;4;64.58
157441;1098;2001;5;0.00
157550;1033;2005;1;97.62
157660;1003;2001;4;0.00
157771;1094;2004;2;100.00
157883;1018;2003;1;20.83
157996;1036;2003;5;60.42
158110;1070;2002;5;0.00
158225;1052;2002;2;100.00
158341;1098;2004;2;100.00
158458;1063;2005;1;100.00
158576;1029;2004;3;79.59
158695;1085;2002;3;62.50
158815;1001;2002;3;100.00
158936;1044;2002;2;77.50
159058;1022;2003;3;0.00
159181;1073;2004;4;0.00
159305;1058;2002;1;100.00
159430;1023;2002;3;12.50
159556;1073;2002;4;0.00
159683;1029;2003;5;72.92
159811;1004;2005;5;100.00
159940;1040;2004;4;0.00
160070;1032;2002;1;0.00
160201;1067;2005;4;40.48
160333;1063;2003;4;0.00
160466;1099;2005;2;76.19
160600;1031;2001;1;100.00
160735;1015;2005;1;80.95
160871;1084;2004;1;100.00
161008;1078;2003;1;22.92
161146;1021;2003;2;0.00
161285;1022;2003;5;100.00
161425;1012;2001;1;76.67
161566;1095;2003;2;2.08
161708;1052;2003;4;0.00
161851;1050;2002;3;100.00
161995;1088;2004;3;61.22
162140;1008;2004;3;0.00
162286;1075;2003;3;91.67
162433;1018;2001;1;50.00
162581;1069;2001;2;80.00
162730;1073;2001;4;0.00
162880;1051;2003;2;62.50
163031;1050;2003;1;100.00
163183;1100;2002;2;0.00
163336;1086;2004;2;73.47
163490;1077;2002;5;0.00
163645;1056;2003;4;100.00
163801;1010;2001;3;83.33
163958;1011;2001;4;60.00
164116;1048;2004;4;59.18
164275;1100;2002;5;15.00
164435;1057;2004;2;0.00
164596;1050;2001;2;0.00
164758;1068;2003;2;0.00
164921;1015;2002;3;60.00
165085;1080;2002;2;0.00
165250;1005;2002;4;100.00
165416;1096;2003;2;25.00
165583;1063;2003;4;56.25
165751;1066;2001;2;90.00
165920;1002;2002;3;15.00
166090;1030;2004;4;0.00
166261;1064;2002;2;0.00
166433;1018;2005;2;100.00
166606;1057;2003;4;100.00
166780;1062;2002;2;100.00
166955;1071;2002;3;100.00
167131;1083;2005;1;0.00
167308;1073;2003;1;68.75
167486;1050;2005;3;100.00
167665;1054;2004;1;4.08
167845;1068;2001;1;100.00
168026;1073;2003;3;4.17
168208;1074;2001;1;43.33
168391;1088;2001;5;96.67
168575;1017;2001;1;0.00
168760;1096;2001;2;0.00
168946;1076;2002;4;55.00
169133;1085;2004;3;46.94
169321;1075;2004;4;0.00
169510;1048;2001;3;70.00
169700;1044;2005;3;11.90
169891;1067;2002;1;100.00
170083;1003;2003;5;68.75
170276;1038;2002;5;0.00
170470;1053;2001;1;100.00
170665;1025;2005;2;0.00
170861;1014;2003;3;100.00
171058;1059;2003;1;100.00
171256;1045;2005;2;14.29
171455;1079;2003;5;2.08
171655;1085;2002;5;100.00
171856;1031;2001;5;50.00
172058;1092;2005;2;0.00
172261;1071;2003;5;100.00
172465;1078;2002;1;35.00
172670;1077;2002;3;100.00
172876;1064;2002;3;12.50
173083;1100;2004;2;100.00
173291;1059;2001;2;66.67
173500;1095;2004;1;100.00
173710;1049;2004;1;0.00
173921;1014;2004;4;4.08
174133;1021;2004;5;100.00
174346;1094;2002;4;85.00
174560;1094;2003;1;0.00
174775;1071;2001;3;6.67
174991;1098;2002;3;100.00
175208;1027;2004;4;40.82
175426;1061;2003;1;100.00
175645;1018;2001;1;63.33
175865;1053;2001;2;40.00
176086;1046;2005;4;100.00
176308;1002;2003;1;95.83
176531;1015;2003;2;0.00
176755;1053;2003;5;0.00
176980;1054;2004;1;28.57
177206;1051;2002;5;35.00
177433;1088;2001;3;13.33
177661;1049;2001;2;20.00
177890;1014;2001;5;53.33
178120;1012;2005;4;54.76
178351;1096;2001;2;100.00
178583;1025;2003;4;100.00
178816;1033;2005;5;95.24
179050;1043;2003;4;43.75
179285;1051;2002;5;97.50
179521;1050;2002;5;17.50
179758;1017;2005;3;100.00
179996;1052;2002;3;95.00
180235;1007;2002;3;30.00
180475;1004;2002;1;100.00
180716;1066;2003;5;100.00
180958;1086;2002;2;100.00
181201;1063;2002;5;47.50
181445;1059;2003;4;100.00
181690;1047;2003;1;56.25
181936;1063;2001;4;63.33
182183;1027;2001;2;100.00
182431;1037;2002;5;40.00
182680;1059;2002;5;80.00
182930;1003;2005;1;100.00
183181;1013;2001;4;73.33
183433;1080;2002;4;40.00
183686;1001;2001;3;0.00
183940;1012;2005;1;0.00
184195;1020;2002;3;0.00
184451;1021;2003;5;100.00
184708;1076;2004;2;0.00
184966;1082;2004;4;20.41
185225;1072;2003;4;54.17
185485;1090;2004;3;30.61
185746;1078;2004;3;0.00
186008;1095;2004;4;46.94
186271;1019;2005;1;0.00
186535;1070;2002;4;100.00
186800;1003;2005;2;0.00
187066;1083;2002;5;0.00
187333;1092;2002;2;100.00
187601;1092;2001;3;100.00
187870;1050;2004;3;67.35
188140;1005;2002;5;92.50
188411;1023;2004;4;0.00
188683;1003;2002;4;0.00
188956;1002;2003;1;75.00
189230;1051;2003;2;72.92
189505;1022;2001;2;0.00
189781;1010;2003;1;22.92
190058;1030;2003;1;64.58
190336;1037;2005;1;19.05
190615;1076;2005;3;0.00
190895;1072;2004;2;4.08
191176;1037;2003;1;31.25
191458;1081;2004;3;36.73
191741;1073;2005;5;88.10
192025;1100;2002;3;22.50
192310;1005;2003;5;0.00
192596;1099;2002;4;90.00
192883;1011;2001;3;0.00
193171;1082;2001;5;96.67
193460;1066;2001;4;0.00
193750;1050;2001;2;46.67
194041;1097;2005;3;42.86
194333;1099;2002;4;0.00
194626;1087;2001;4;0.00
194920;1088;2003;4;0.00
195215;1086;2003;2;0.00
195511;1087;2001;4;96.67
195808;1059;2002;2;95.00
196106;1093;2002;3;0.00
196405;1018;2005;2;100.00
196705;1014;2003;2;0.00
197006;1057;2005;1;100.00
197308;1063;2005;5;2.38
197611;1084;2001;5;10.00
197915;1074;2003;4;100.00
198220;1083;2003;4;0.00
198526;1014;2005;4;100.00
198833;1037;2003;5;39.58
199141;1085;2004;2;100.00
199450;1078;2003;2;0.00
199760;1048;2002;4;77.50
200071;1022;2004;4;69.39
200383;1072;2005;4;57.14
200696;1035;2004;3;53.06
201010;1025;2005;2;0.00
201325;1045;2002;1;100.00
201641;1085;2003;5;100.00
201958;1022;2002;2;100.00
202276;1008;2002;3;30.00
202595;1078;2005;4;0.00
202915;1047;2001;5;70.00
203236;1022;2001;2;0.00
203558;1031;2005;4;0.00
203881;1087;2003;4;54.17
204205;1053;2004;1;14.29
204530;1089;2004;1;0.00
204856;1057;2002;3;27.50
205183;1023;2003;4;100.00
205511;1034;2004;4;100.00
205840;1048;2001;2;86.67
206170;1099;2001;5;0.00
206501;1049;2002;4;100.00
206833;1026;2004;5;0.00
207166;1056;2005;3;0.00
207500;1094;2003;4;100.00
207835;1005;2003;3;0.00
208171;1054;2005;3;100.00
208508;1089;2003;1;97.92
208846;1089;2004;2;100.00
209185;1053;2002;5;65.00
209525;1094;2003;1;100.00
209866;1074;2001;3;60.00
210208;1100;2003;1;33.33
210551;1041;2004;2;36.73
210895;1070;2001;1;0.00
211240;1022;2003;3;72.92
211586;1030;2002;1;82.50
211933;1095;2003;4;0.00
212281;1003;2002;5;62.50
212630;1060;2001;3;60.00
212980;1033;2001;4;26.67
213331;1098;2004;5;100.00
213683;1046;2001;4;86.67
214036;1042;2005;4;38.10
214390;1073;2002;3;92.50
214745;1022;2002;1;80.00
215101;1020;2004;4;100.00
215458;1013;2004;3;100.00
215816;1076;2002;4;45.00
216175;1080;2002;5;25.00
216535;1074;2002;5;100.00
216896;1034;2003;5;0.00
217258;1042;2002;3;100.00
217621;1038;2003;3;20.83
217985;1062;2003;3;95.83
218350;1089;2003;2;100.00
218716;1017;2001;4;0.00
219083;1024;2003;4;100.00
219451;1099;2005;4;2.38
219820;1084;2001;2;53.33
220190;1002;2005;5;2.38
220561;1019;2002;5;0.00
220933;1044;2002;3;100.00
221306;1032;2004;4;95.92
221680;1017;2002;2;92.50
222055;1001;2003;5;37.50
222431;1036;2002;1;37.50
222808;1046;2005;3;100.00
223186;1099;2002;4;55.00
223565;1053;2002;3;100.00
223945;1098;2002;1;65.00
224326;1071;2002;1;50.00
224708;1099;2003;5;0.00
225091;1015;2005;4;0.00
225475;1053;2002;3;0.00
225860;1040;2005;3;0.00
226246;1076;2005;2;100.00
226633;1073;2001;5;100.00
227021;1097;2005;2;0.00
227410;1037;2002;1;15.00
227800;1092;2005;5;0.00
228191;1011;2004;4;51.02
228583;1075;2004;2;97.96
228976;1032;2002;4;100.00
229370;1022;2004;1;0.00
229765;1043;2003;2;100.00
230161;1080;2005;1;0.00
230558;1094;2004;4;32.65
230956;1078;2001;5;0.00
231355;1068;2001;4;76.67
231755;1093;2002;5;17.50
232156;1036;2005;2;0.00
232558;1039;2004;5;57.14
232961;1081;2005;1;16.67
233365;1043;2004;1;40.82
233770;1087;2004;4;95.92
234176;1095;2002;3;100.00
234583;1066;2004;2;0.00
234991;1007;2002;2;100.00
235400;1096;2001;5;100.00
235810;1098;2001;1;0.00
236221;1089;2005;1;54.76
236633;1086;2001;5;0.00
237046;1051;2005;4;69.05
237460;1099;2001;3;60.00
237875;1049;2003;1;100.00
238291;1034;2003;5;8.33
238708;1086;2003;4;0.00
239126;1080;2002;2;100.00
239545;1098;2005;4;100.00
239965;1018;2003;4;0.00
240386;1030;2005;2;92.86
240808;1056;2005;2;42.86
241231;1078;2001;3;6.67
241655;1009;2005;3;66.67
242080;1064;2005;3;0.00
242506;1015;2002;5;37.50
242933;1057;2004;5;100.00
243361;1094;2002;2;67.50
243790;1084;2003;4;0.00
244220;1076;2003;1;100.00
244651;1068;2005;2;80.95
245083;1048;2001;2;60.00
245516;1005;2003;4;87.50
245950;1073;2004;2;95.92
246385;1020;2003;1;0.00
246821;1026;2002;3;100.00
247258;1050;2003;3;100.00
247696;1038;2001;3;100.00
248135;1084;2005;1;83.33
248575;1060;2005;3;100.00
249016;1070;2002;2;82.50
249458;1086;2005;2;14.29
249901;1022;2005;2;0.00
250345;1059;2002;4;50.00
250790;1080;2001;1;53.33
251236;1031;2002;2;30.00
251683;1013;2001;1;63.33
252131;1036;2005;3;45.24
252580;1018;2004;1;0.00
253030;1065;2005;4;47.62
253481;1040;2002;4;42.50
253933;1012;2001;4;100.00
254386;1096;2003;3;12.50
254840;1010;2005;4;0.00
255295;1087;2001;3;13.33
255751;1036;2004;3;77.55
256208;1025;2004;2;0.00
256666;1002;2003;1;0.00
257125;1076;2001;5;40.00
257585;1018;2001;1;23.33
258046;1064;2001;2;0.00
258508;1067;2003;2;2.08
258971;1062;2005;3;69.05
259435;1036;2003;4;100.00
259900;1068;2004;2;100.00
260366;1069;2003;4;12.50
260833;1009;2003;2;0.00
261301;1099;2003;1;0.00
261770;1039;2001;2;36.67
262240;1038;2002;1;12.50
262711;1087;2001;1;66.67
263183;1029;2001;5;100.00
263656;1061;2005;4;100.00
264130;1082;2005;3;100.00
264605;1023;2001;1;80.00
265081;1005;2005;2;0.00
265558;1053;2004;4;0.00
266036;1084;2002;5;92.50
266515;1081;2003;3;100.00
