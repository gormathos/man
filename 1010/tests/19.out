Pascal: 63 93 23 23
Python: 62 101 24 31
C++: 67 102 34 18
Java: 68 103 24 29
C#: 54 81 22 21
